src/node/http-request.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  HttpRequest,
+  HttpMethod,
+  HttpBody,
+  HttpRequestHeaders,
+  HttpQuery
+} from "../http";
+import { ParsedUrl, ParsedQuery, parseUrl, stringify } from "query-string";
```

Comment:
You can use the native `URLSearchParams` class to handle querystrings. Although if you use `URL` class, it is not needed anymore.